### DISCUSSION TITLE
Filters feedback

### DISCUSSION THREAD

**Discussion by adrianthedev** [avo-hq/avo#838] [Feature feedback]
Let's use this space to provide feedback for the Filters feature.

Post [an issue](https://github.com/avo-hq/avo/issues/new/choose) if something is broken.

**Comment by ckmorris** [comment]
The filters feedback page actually links to the fields feedback page. I had to go to the index in order to find this page. 

It would be very helpful to be able to access the `current_user` object within the Filtering framework. Additionally, it would be useful to be able to conditionally display filters based on the `current_user` object. `filter SomeFilter` and then link that to a Pundit Policy method or do it like the Actions with `self.visible = -> (resource:) { }` so that we can grab the context and the `current_user` object.

**Reply by adrianthedev** [reply]
This is good feedback! So have `self.visible` on the filter class, similar to how we do it in [actions](https://docs.avohq.io/2.0/actions.html#actions-visibility).

**Reply by adrianthedev** [reply]
Let's track it in https://github.com/avo-hq/avo/issues/1253
